src/pushInfo/PushInfo.style.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components'
+
+import { color, font, fontWeight, radius, space } from '../_utils/branding'
+
+export const animationDuration = 700
```

Comment:
IMO it is a good practice to declare animations with style, since it's more a UI thing.. 
So I'm ok for declaring here.